pkg/keb/clusterType.go

```diff
@@ -1,4 +1,9 @@
-{
+package keb
+
+type TestCluster []byte
+
+var (
+	Production TestCluster = []byte(`{
```

Comment:
Same here, this one is also only for testing, and we can write a function to read the content directly from json file. but now leave it like this is fine, let's refactor later.